server/src/server.ts

```diff
@@ -375,19 +376,37 @@ function onMessage(msg: m.Message) {
         // error: code and message set in case an exception happens during the definition request.
       };
       send(definitionResponse);
+    } else if (msg.method === p.ReferencesRequest.method) {
+      // https://microsoft.github.io/language-server-protocol/specifications/specification-current/#textDocument_references
+      let result: Location | null = utils.runAnalysisAfterSanityCheck(
+        msg,
+        (filePath) => [
+          "references",
+          filePath,
+          msg.params.position.line,
+          msg.params.position.character,
+        ]
+      );
+      let definitionResponse: m.ResponseMessage = {
+        jsonrpc: c.jsonrpcVersion,
+        id: msg.id,
+        result,
+        // error: code and message set in case an exception happens during the definition request.
+      };
+      send(definitionResponse);
     } else if (msg.method === p.CompletionRequest.method) {
       let code = getOpenedFileContent(msg.params.textDocument.uri);
       let tmpname = utils.createFileInTempDir();
       fs.writeFileSync(tmpname, code, { encoding: "utf-8" });
       let result:
         | CompletionItem[]
         | null = utils.runAnalysisAfterSanityCheck(msg, (filePath) => [
-          "complete",
-          filePath,
-          msg.params.position.line,
-          msg.params.position.character,
-          tmpname,
-        ]);
+        "complete",
```

Comment:
These should probably be put back

Reply:
This is just auto formatting. Should I revert the formatting?

Reply:
Ah no then. I use Prettier because @IwanKaramazow's editor was configured to use it. Nvm